packages/xo-server/sample.config.toml

```diff
@@ -95,6 +95,12 @@ port = 80
 # #hostname = '127.0.0.1'
 # port = 443
 #
+# # Whether to autogenerate a self signed certificate if the `cert` and `file`
+# # entries could not be found.
+# #
+# # Default: false
```

Comment:
Changing the default to `true` would make it easier to use with existing appliances.

Do you see any issues that could happened with this new default?

Reply:
I agree with make it true to force the `HTTPS` connection.

Reply:
This does not force HTTPS, because it only works with an HTTPS config (one with `cert` and `key` settings).

But it changes the behavior by generating the cert and key files in case they are missing.

Before enabling this by default, I'm trying to thinking about problems it can causes.

Reply:
So:
- for users that didn't put `cert`/`key` in their config, it won't change anything
- for users that already had their own certificate configured, it won't change anything either
- for new users, if they put `cert`/`key` in their config without creating the files, it will generate a certificate for them

I don't see any problem with that either.

Reply:
Thank you for your answers, I've changed the default value accordingly.